### DISCUSSION TITLE
What's the purpose of  i18n folder and more

### DISCUSSION THREAD

**Discussion by 96312** [Lednerb/bilberry-hugo-theme#622] [Q&A] [answered]
Hi, I'm trying to learn hugo (multilingual) and wanted to create a very basic blog with 2 languages. 

But I don't just want to copy this template, I want to know why something was done? So I'm referring official docs and this theme to learn.

Can you please tell why the folder i18n has so many .toml files when you clearly provided both translations like [here](https://github.com/Lednerb/bilberry-hugo-theme/tree/master/exampleSite/content/audio)?

**Comment by Sp-cy** [comment]
My understanding is that the i18n folder is where Site wide translations can be set up for words in the Menu or Footer. One thing is the article or post translation and another the site translation.

**Reply by Sp-cy** [reply, marked as answer]
I made my AI chat assistant give a better explanation:

> The `i18n` folder in Hugo is used to store localized versions of your website. It contains subfolders for each language that you want to support, and within those subfolders are the translated content files.
> 
> When a user visits your website with a specific language code (e.g. `en`, `fr`, `de`), Hugo will look in the `i18n` folder for the corresponding language's version of the content file, and serve that to the user instead of the default English version. This allows you to create a multilingual website that is accessible to users who speak different languages.
> 
> For example, if your website has a page called "About Us", you might have an `i18n/en` folder with a file called "about-us.md" (or "about-us.html") that contains the English version of the content, and an `i18n/fr` folder with a file called "a-propos.md" (or "a-propos.html") that contains the French version of the content. When a user visits your website with the language code `en`, Hugo will serve the English version of the content, and when a user visits your website with the language code `fr`, Hugo will serve the French version of the content.
> 
> It's worth noting that the `i18n` folder is only used for static content, such as pages and posts. If you have dynamic content (e.g. blog posts) that needs to be translated, you may need to use a different approach.

Hope this helps.

**Reply by 96312** [reply]
Thank you for the explanation! I got it :)

**Comment by igor-baiborodine** [comment]
Discussions, as opposed to issues, should not be closed; therefore, I'm reopening it.
